api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -128,6 +128,11 @@ func (in *XlineClusterSpec) DeepCopyInto(out *XlineClusterSpec) {
 		*out = new(string)
 		**out = **in
 	}
+	if in.Args != nil {
+		in, out := &in.Args, &out.Args
+		*out = make([]string, len(*in))
+		copy(*out, *in)
```

Comment:
Can it be changed to this? Why do we need two layers of pointers here?
```go
in, out := in.Args, out.Args
out = make([]string, len(in))
copy(out, in)
```

Reply:
This file is generated, I cannot understand it either.
But it works.